### PR TITLE
CROSSING-7064; version of docker installed is 17.05.0-ce

### PR DESCRIPTION
**Pull request by shazChaudhry**
CROSSING-7064

- Docker role is passing
- The ansible role will install docker v17.06.0 ce

### CODE REVIEW

File: tasks/Debian.yml [hunk 1]
```diff
@@ -7,9 +7,10 @@
     - "linux-image-extra-virtual-lts-{{ ansible_distribution_release }}"
     - apt-transport-https
```
Comment (sirv1ne) [inline tasks/Debian.yml]: Could you remove these old dependencies please! They don't seem to be required anymore so should be cleaned up :)

File: tasks/Debian.yml [hunk 2]
```diff
@@ -7,9 +7,10 @@
     - "linux-image-extra-virtual-lts-{{ ansible_distribution_release }}"
```
Comment (sirv1ne) [inline tasks/Debian.yml]: Could you remove these old dependencies please! They don't seem to be required anymore so should be cleaned up :)

**Review by sirv1ne — CHANGES_REQUESTED** [review]

File: tasks/Debian.yml (hunk 2 above)
Comment (sirv1ne) [reply tasks/Debian.yml]: Packages to go are: "linux-image-extra-virtual", "linux-image-extra-virtual-lts-{{ ansible_distribution_release }}", "python", "python-pip", "python-httplib2". None of these are listed as dependencies in https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/#install-using-the-repository.

**Comment by shazChaudhry** [comment]
Removed packages not required or used anywhere else within the role for docker installation. As discussed, curl package is not needed and I have left "python, python-pip, python-httplib2" alone as presence for some of the modules is checked in the main.yml

**Comment by sirv1ne** [comment]
LGTM

**Review by sirv1ne — APPROVED** [review]

**Pull request merged**